static-site/src/pages/users.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "styled-components";
+import UserDataWrapper from "../layouts/userDataWrapper";
+import NavBar from '../components/nav-bar';
+import { Logos } from "../components/logos";
+
+const UserPage = (props) => {
+  const { user } = props;
+
+  const LoggedIn = () => (
+    <p>
+      You&apos;re logged in as <strong>{user.username}</strong>.<br />
+      You have access to the following groups:
+      <UserGroupsList>
+        {user.groups.map((group) => (
+          <li>
+            <a href={`/groups/${group}`}>{group}</a>
+          </li>
+        ))}
+      </UserGroupsList>
+      <a href="/logout">Logout</a>
+    </p>
+  );
+
+  const LoggedOut = () => (
+    <p>
+      You are not logged in.<br />
+      <a href="/login">Login</a>
+    </p>
+  );
+
+  return (
+    <UserContainer>
+      {user
+        ? LoggedIn()
+        : LoggedOut()
+      }
+    </UserContainer>
+  );
+};
+
+const UserContainer = styled.div`
+  max-width: 640px;
+  padding: 130px 0px 80px 0px;
+  margin-top: 0px;
+  margin-right: auto;
+  margin-bottom: 0px;
+  margin-left: auto;
+  text-align: center;
+  font-size: 28px;
+  font-weight: 300;
+  line-height: ${(props) => 1.4 * props.theme.niceLineHeight};
+`;
+
+const UserGroupsList = styled.ul`
+  display: table;
+  margin: 0 auto !important;
+`;
+
+const Users = () => {
+  return (
+    <div className="index-container">
+      <main>
+        <UserDataWrapper>
+          <NavBar minified/>
```

Comment:
Please remove the `minified` prop here as this will add in the nextstrain logo which matches the splash page